client/app/components/packages/pas-form/edit.hbs

```diff
@@ -398,17 +398,18 @@
           onClick={{fn this.updateAttr @package.pasForm "dcpRestrictivedeclaration" false}}
           data-test-dcprestrictivedeclaration-no
         /><label for="dcprestrictivedeclaration-no">No</label>
+        {{#if (eq @package.pasForm.dcpRestrictivedeclaration true)}}
```

Comment:
This is defined as a string in the model. I worry that the conditional won't work unless we make sure it is a real boolean and that CRM considers it a boolean.

Reply:
Happy to do this but can't suggest a commit here 

```
  @attr('string')
  dcpRestrictivedeclaration;
```

W just need that to be a boolean:

```
  @attr('boolean')
  dcpRestrictivedeclaration;
```

Reply:
Thanks I'll make that update

Reply:
Actually I think it's okay to leave it as is? This question should work the same as `dcpProjectareaindustrialbusinesszone` and `dcpProjectareaindutrialzonename` which appear to be working correctly. 

[`dcpProjectareaindustrialbusinesszone` is saved as "string" on the model](https://github.com/NYCPlanning/labs-applicant-portal/blob/develop/client/app/models/pas-form.js#L128), and the template logic uses [`eq @package.pasForm.dcpProjectareaindustrialbusinesszone true`](https://github.com/NYCPlanning/labs-applicant-portal/blob/develop/client/app/components/packages/pas-form/edit.hbs#L272)

Reply:
Ah then those are wrong because they will get serialized as strings.

We'll need to fix those as well.

Reply:
Oops, also forgot to mention, you cna also just say `   {{#if @package.pasForm.dcpRestrictivedeclaration}}`